### PR TITLE
#463: Added user install script for gatk ...

### PR DESCRIPTION
**Pull request by abaumann**
...that can be used on cluster initialization.  Does not deal with any python packages that are needed for certain tools, only deals with the jars.

\<your comments for this PR go here\>

Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get two thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: jupyter-docker/user-scripts/install_gatk.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+GATK_ZIP_PATH=/tmp/gatk-$GATK_VERSION.zip
+
+#download the gatk zip if it doesn't already exist
+if ! [ -f $GATK_ZIP_PATH ]; then
+  # curl and follow redirects and output to a temp file
+  curl -L -o $GATK_ZIP_PATH https://github.com/broadinstitute/gatk/releases/download/$GATK_VERSION/gatk-$GATK_VERSION.zip
+fi
+
+# unzip with forced overwrite (if necessary) to /bin
+unzip -o $GATK_ZIP_PATH -d /bin/
```
Comment (rtitle) [inline jupyter-docker/user-scripts/install_gatk.sh]: Nit: `/bin/` might not be the best place to install the full gatk distribution. How about unzipping it to `/etc/` and then symlinking `/etc/gatk-$GATK_VERSION/gatk -> /bin/gatk`?

File: jupyter-docker/user-scripts/install_gatk.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+# make a symlink to gatk right inside bin so it's available from the existing PATH
+ln -s /bin/gatk-4.0.6.0/gatk /bin/gatk
```
Comment (rtitle) [inline jupyter-docker/user-scripts/install_gatk.sh]: Could use the `$GATK_VERSION` variable here

**Comment by abaumann** [comment]
Good catch on version - i also changed to installing into /etc and symlinking to that in /bin like suggested

**Review by rtitle — APPROVED** [review]
:shipit:! Pending verification from Tiffany.

**Comment by rtitle** [comment]
Weird Jenkins build error - jenkins retest

**Pull request merged**